engine/schema/utils.ts

```diff
@@ -167,11 +161,20 @@ function isQuotaExceededError(err: unknown): boolean {
       err.name === "NS_ERROR_DOM_QUOTA_REACHED")
   );
 }
+
 export const denoDoc = async (
   path: string,
-  importMap?: string,
+  _importMap?: string,
 ): Promise<DocNode[]> => {
   try {
+    const docCacheKey = path;
+    if (context.isDeploy) {
+      const cached = denoDocLocalCache[docCacheKey];
+      if (!cached) {
+        throw new Error(`could not resolve ${docCacheKey} on denodoc`);
```

Comment:
Offer a solution for the probably error cause.

```suggestion
        throw new Error(`could not resolve ${docCacheKey} on denodoc`); Create denodoc cache locally with "deno task doccache" and push the denocache.zst file.
```

Reply:
Do we have a task to create the deno doc cache?

Reply:
yes. deno task start --gen-only